src/prefect/indicators/util.py

```diff
@@ -0,0 +1,61 @@
+"""QualiCharge prefect indicators: util.
+
+Common indicators functions and constants.
+"""
+from .models import Level
+
+POWER_RANGE = """
+    puissance(category, p_cat) AS (
+        VALUES
+            (numrange(0, 15.0), 1),
+            (numrange(15.0, 26.0), 2),
+            (numrange(26, 65.0), 3),
+            (numrange(65, 175.0), 4),
+            (numrange(175, 360.0), 5),
+            (numrange(360, NULL), 6)
+    )"""
+
+def params_level(level):
+    """Get level_id and join_level query parameters."""
+    match level:
+        case Level.CITY:
+            return {
+                    "level_id": "City.id",
+                    "join_level": ""
+                   }
+        case Level.EPCI:
+            return {
+                    "level_id": "EPCI.id",
+                    "join_level": "INNER JOIN EPCI ON City.epci_id = EPCI.id"
+                   }
+        case Level.DEPARTMENT:
+            return {
+                    "level_id": "Department.id",
+                    "join_level": """
+                    INNER JOIN Department ON City.department_id = Department.id
+                    """
+                   }
+        case Level.REGION:
+            return {
+                    "level_id": "Region.id",
+                    "join_level": """
+                    INNER JOIN Department ON City.department_id = Department.id
+                    INNER JOIN Region ON Department.region_id = Region.id
+                    """
+                   }
+        case _:
+            raise NotImplementedError("Unsupported level %d", level)
+
+def table_level(level):
+    """Get table name for a level."""
+    match level:
+        case Level.CITY:
+            return "city"
+        case Level.EPCI:
+            return "epci"
+        case Level.DEPARTMENT:
+            return "department"
+        case Level.REGION:
+            return "region"
+        case _:
+            raise NotImplementedError("Unsupported level %d", level)
```

Comment:
As `level` is now typed:

```suggestion
    return level.name.lower()
```

Reply:
It's so simple that I'm not sure it's required to define this utility.

Reply:
We rather should instead raise an error for the `Level.NATIONAL` case.

Reply:
Function is removed and replace by  : 
```python
@task(task_run_name="targets-for-level-{level:02d}")
def get_targets_for_level(connection: Connection, level: Level) -> pd.DataFrame:
    """Get registered targets for level from QualiCharge database."""
    if level == Level.NATIONAL:
         raise NotImplementedError("Unsupported level %d", level)
    return pd.read_sql_table(level.name.lower(), con=connection)
```